rx-extensions/src/test/java/com/appunite/rx/functions/Functions1Test.java

```diff
@@ -0,0 +1,62 @@
+package com.appunite.rx.functions;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import rx.Observer;
+import rx.subjects.PublishSubject;
+
+import static org.mockito.Mockito.verify;
+
+public class Functions1Test {
+
+    private PublishSubject<Object> subject;
+    @Mock
+    Observer<String> observer;
+
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
+
+        subject = PublishSubject.create();
+        subject.map(Functions1.toStringFunction()).subscribe(observer);
```

Comment:
for the future is much better to use TestSubscriber instead of mock observer, but this is ok too